### PR TITLE
Feature/Adds queries for twitter

### PR DESCRIPTION
**Pull request by jaimeyoon-yje**

### CODE REVIEW

**Comment by jaimeyoon-yje** [comment]
[Twitter Reliability & Usage Growth [3]](https://trello.com/c/hJgUvX1p/4693-twitter-reliability-usage-growth-3)

File: src/main/resources/job-scheduler-config.json [hunk 1]
```diff
@@ -76,6 +76,10 @@
       "destinationTableId": "RssUsageByMonth",
       "fileName": "queries/rss-widget-usage-by-month.sql"
     },
+    {
+      "destinationTableId": "TwitterUsageByMonth",
+      "fileName": "queries/twitter-widget-usage-by-month.sql"
```
Comment (rudibravo) [inline src/main/resources/job-scheduler-config.json]: @jungeunyoon do you need to commit those files as well?

**Review by rudibravo — COMMENTED** [review]
@jungeunyoon I have a question inline. Thanks

File: src/main/resources/job-scheduler-config.json (hunk 1 above)
Comment (rudibravo) [reply src/main/resources/job-scheduler-config.json]: @jungeunyoon nevermind, I see them now.

**Review by rudibravo — APPROVED** [review]
@jungeunyoon looks good 👍

File: src/main/resources/job-scheduler-config.json (hunk 1 above)
Comment (jaimeyoon-yje) [reply src/main/resources/job-scheduler-config.json]: @rudibravo actually I caught a typo from that comment. thanks. `queries/twitter-widget-usage-by-month.sql` --> `queries/twitter-component-usage-by-month.sql`

**Pull request merged**